tests/features/language-switcher.feature

```diff
@@ -18,8 +18,8 @@ Feature: Language selection
       | Gaeilge     |
       | hrvatski    |
       | italiano    |
-      | latviešu    |
```

Comment:
This change is incorrect, but it made us discover that we have the wrong weight associated to languages. Latvian needs to be before Lithuanian, so we need to update the weight config. Please also keep an eye on other languages based on ec.europa.eu.

Reply:
The list doesn't have a good order. see the Spanish button on https://ec.europa.eu/info/ 
Please create a follow-up ticket to change the order according to the business.